packages/grant-explorer/src/features/projects/ViewProject.tsx

```diff
@@ -0,0 +1,296 @@
+  return (
+    <svg
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M4 0C3.44772 0 3 0.447715 3 1V2H2C0.895431 2 0 2.89543 0 4V14C0 15.1046 0.895431 16 2 16H14C15.1046 16 16 15.1046 16 14V4C16 2.89543 15.1046 2 14 2H13V1C13 0.447715 12.5523 0 12 0C11.4477 0 11 0.447715 11 1V2H5V1C5 0.447715 4.55228 0 4 0ZM4 5C3.44772 5 3 5.44772 3 6C3 6.55228 3.44772 7 4 7H12C12.5523 7 13 6.55228 13 6C13 5.44772 12.5523 5 12 5H4Z"
+        fill="currentColor"
+      />
+    </svg>
+  );
+};
+
+export default function ViewProject() {
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  const { projectId } = useParams();
+
+  const dataLayer = useDataLayer();
+
+  const { data, error } = useProject(
+    {
+      projectId: projectId,
+    },
+    dataLayer
+  );
+
+  const project = data?.project;
+
+  const breadCrumbs = [
+    {
+      name: "Explorer Home",
+      path: "/",
+    },
+    {
+      name: "Projects",
+      path: `/projects/`,
+    },
+    {
+      name: project?.metadata.title,
+      path: `/projects/${projectId}`,
+    },
+  ] as BreadcrumbItem[];
+
+  const {
+    metadata: { title, description = "", bannerImg },
+  } = project ?? { metadata: {} };
+  const projectDetailsTabs = useMemo(
+    () => [
+      {
+        name: "Project details",
+        content: (
+          <>
+            <h3 className="text-3xl mt-8 mb-4 font-modern-era-medium text-blue-800">
+              About
+            </h3>
+            {project ? (
+              <>
+                <Detail text={description} testID="project-metadata" />
+              </>
+            ) : (
+              <SkeletonText />
+            )}
+          </>
+        ),
+      },
+    ],
+    [project, description]
+  );
+
+  const handleTabChange = (tabIndex: number) => {
+    setSelectedTab(tabIndex);
+  };
+
+  return (
+    <>
+      <DefaultLayout>
+        <div className="flex flex-row justify-between my-8">
+          <div className="flex items-center pt-2" data-testid="bread-crumbs">
+            <Breadcrumb items={breadCrumbs} />
+          </div>
+        </div>
+        <div className="mb-4">
+          <ProjectBanner
+            bannerImgCid={bannerImg ?? null}
+            classNameOverride="h-32 w-full object-cover lg:h-80 rounded md:rounded-3xl"
+            resizeHeight={320}
+          />
+          <div className="pl-4 sm:pl-6 lg:pl-8">
+            <div className="sm:flex sm:items-end sm:space-x-5">
+              <div className="flex">
+                <ProjectLogo {...project?.metadata} />
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="md:flex gap-4 flex-row-reverse">
+          <div className="flex-1">
+            {error === undefined ? (
+              <>
+                <Skeleton isLoaded={Boolean(title)}>
```

Comment:
`title !== undefined`

Reply:
actually maybe it's better to check that `data !== undefined && error === undefined`